CONTRIBUTING.md

```diff
@@ -22,20 +22,43 @@ To make a change to the FVM.
 
 ## Releasing
 
-To prepare a release.
-
-1. Make sure the crate's `CHANGELOG.md` is up-to-date (look through `git log -- path/to/crate`).
-2. Make sure the changes are well tested. See [builtin actors testing][builtin actors testing].
-3. Pick a next version (for each crate you're releasing). Look through the changelogs and carefully
-   decide if it's a breaking release or not. Then read [rust's semver
-   documentation](https://doc.rust-lang.org/cargo/reference/semver.html).
-4. Install `cargo-edit` (`cargo install cargo-edit`).
-5. Create a new branch.
-6. Use `cargo set-version` to set the version for each crate you're releasing. This will both update
-   the crate version, and make all other crates in the workspace depend on the latest version.
-7. Make sure the changelogs are all up-to-date, set the release date & version, and add a new
-   "Unreleased" section.
-8. Commit your changes, and make a PR.
+The primary crates, `fvm`, `fvm_shared`, `fvm_sdk`, and `fvm_integration_tests`, are versioned and
+released together. Other crates in the workspace are versioned and released independently.
+
+Currently version numbering is not strictly semver compatible. Breaking changes may be included in
+minor releases while major releases are reserved for dropping old network versions.
+
+All changes should be well tested. See [builtin actors testing][builtin actors testing].
+
+### Primary FVM crates
+
+To propose a new release, open a pull request with the following changes:
+
+1. Update the version in `Cargo.toml`: `workspace.package→version`.
+2. Update the version of the coupled workspace dependencies in `Cargo.toml` to match the new version
+   (leaving semver range specifier `~` intact):
+   1. `wokspace.package→fvm→version`
+   2. `wokspace.package→fvm_shared→version`
+   3. `wokspace.package→fvm_sdk→version`
+   4. `wokspace.package→fvm_integration_tests→version`
+3. Update the lockfile with a rebuild: `cargo build`.
```

Comment:
`cargo check --all` should be sufficient (but `cargo build` may not be because it doesn't build everything? I'm not actually sure).